src/main/java/gym/crm/service/impl/TraineeServiceImpl.java

```diff
@@ -56,42 +67,108 @@ public ApiResponse<Void> update(String username, TraineeRequest traineeRequest)
     }
 
     @Override
+    @Transactional
+    public ApiResponse<Void> updatePassword(String username, String oldPassword, String newPassword) {
```

Comment:
ApiResponse not sure that we need this class for me better to have some kind of ErrorResponse class which we build in exceptional handler and just simple POJO object for success response